### DISCUSSION TITLE
Webpack 5 with Next.js seems to break Serverless Function size

### DISCUSSION THREAD

**Discussion by thesandlord** [vercel/vercel#5604] [General]
Hey folks, I'm deploying a Next.js app to Vercel and getting the following message:

```
Error: Required lambda files exceed max lambda size of 50000000 bytes
```

This is surprising to me as I don't have any heavy dependencies. However, I am using Webpack 5 via resolutions in my package.json

```
...
  },
  "resolutions": {
    "webpack": "^5.4.0"
  }
}
```

Removing this and using the default Webpack 4 seems to work fine. No idea why this is happening, and not sure if this is a Next.js issue or a Vercel issue.

Posting here in case anyone else runs into this issue or has suggestions.

**Comment by elibenton** [comment]
I have run into what is likely the same issue.

Using Webpack 4 or 5 doesn't seem to make the difference for me, however, at least not using "resolutions".

Here is a link to my issue:

https://github.com/vercel/next.js/issues/20933#issue-782753097
